docs/getting-started/using-the-block-explorer.md

```diff
@@ -4,59 +4,69 @@
 
 ## ![](../img/dcr-icons/Info.svg){ .dcr-icon } Overview
 
-All blocks and transactions on the Decred blockchain
-are visible through the use of the block explorer, [dcrdata](https://github.com/decred/dcrdata).
+All blocks and transactions on the Decred blockchain are visible through the
+use of the block explorer, [dcrdata](https://github.com/decred/dcrdata).
 
 Public instances of dcrdata are available for the following networks:
 
 - [mainnet](https://dcrdata.decred.org)
 - [testnet](https://testnet.decred.org)
 
-Below is a quick review of some of the information on them.
+---
+
+## ![](../img/dcr-icons/Blocks.svg){ .dcr-icon } Blocks
+
+Blocks can be found by searching for their block height number, clicking on a
+`Height` value from the home page, or from their `BlockHash` value. Older
+blocks will have lower block numbers.
 
-Option         | Explanation
----            | ---
-`Height`       | The block number.
-`Age`          | How long ago the block was added to the blockchain.
-`Transactions` | The number of transactions included in the block.
-`Votes`        | The number of proof-of-stake votes included in the block.
-`Fresh Stake`  | The number of new tickets purchased in this block.
-`Size`         | The size (in bytes) of the block.
+Each block is packed with information. There are two main sections inside a
+block: 
 
-Under `Latest Transactions`, you can see the transaction ID (txid) and
-the value (in DCR) transmitted across the network.
+1. **The Block Header**
+    - This shows summary information about this specific block, as well as a
+    link to the previous block in the chain.
+2. **Transactions**
+    - Consisting of:
+        - The Proof-of-Work Reward
+        - The Treasury Reward
+        - Proof-of-Stake Votes and Rewards
+        - Tickets which will enter the stakepool
```

Comment:
"stakepool" not really the correct term here, "ticket pool" would be more accurate.

tbh I'd just go with "Proof-of-Stake ticket purchases".